.travis.yml

```diff
@@ -30,7 +26,6 @@ matrix:
     - rvm: ruby-head
     - gemfile: gemfiles/rails_6.0_beta.gemfile
 gemfile:
-  - gemfiles/rails_4.2_stable.gemfile
```

Comment:
Removed this since Rails 4.2 doesn't support Bundler 2 and Travis CI installs bundler 2 by default. Rails 4.2 only receives severe security updates at this point, and will be officially EOL once Rails 6 is out. 

Thoughts?

Reply:
I'm fine dropping Rails 4.2 *after* it's officially EOL'd.  That probably won't be too long now considering that Rails 6.0 is in beta.